src/components/NavbarStudy.scss

```diff
@@ -0,0 +1,163 @@
+$primary-color: rgb(82, 105, 207);
+$white-color: #fff;
+$content-color: #1B1B1F;
+
+.menu {
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    top: 50px;
+    left: 0;
+}
+
+.blur {
+    position: absolute;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    backdrop-filter: none;
+    transition: .2s;
+}
+
+.blur--active {
+    position: absolute;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    backdrop-filter: blur(2px);
+    transition: .2s;
+}
+
+.menu__content {
+    position: relative;
+    width: 64px;
+    height: 794px;
+    background-color: $white-color;
+    border-top: 1px solid rgba(0, 0, 0, 0.2);
```

Comment:
Optionally: 
you can create sass mixin with 

border-top: 1px solid rgba(0, 0, 0, 0.2);
border-bottom: 1px solid rgba(0, 0, 0, 0.2);
    border-right: 1px solid rgba(0, 0, 0, 0.2);
    ....
    
    as you often use same properties here and in the file
    src/components/NavbarStudio.scss

Reply:
Now I only have two places where do I use it? so i think there is no reason to use mixin

Reply:
Yes, it is optional